index.html

```diff
@@ -31,11 +33,12 @@ <h2>Which type of message?</h2>
         <br>
         <br>
         <div class="box spacing meditate-box">
-          <img
+          <img id="meditate-bell"
```

Comment:
This will definitely be specific to wherever you work and totally up to the team, but I believe Turing teaches refraining from hardcoding HTML stylings inside the element attributes. It's definitely a GO for initial layout, but worth refactoring into the CSS file after you've figured out what you'd like